Dockerfile

```diff
@@ -1,6 +1,12 @@
 #--- Build stage
 FROM golang:1.19-alpine3.16 AS go-builder
 
+# install Linux header and build-base for native C codes
+RUN apk add --no-cache \
+    ca-certificates \
+    build-base \
+    linux-headers
```

Comment:
`ca-certificates` and `build-base` packages are already installed lower, I propose to add `linux-headers` with them and remove this `RUN` instruction.

Regarding the `linux-headers` package please reference its version (i.e. `linux-headers=5.16.7-r1`), the linter will appreciate 😉